cgroups/fs/apply_raw.go

```diff
@@ -118,21 +141,21 @@ func GetStats(systemPaths map[string]string) (*cgroups.Stats, error) {
 
 // Freeze toggles the container's freezer cgroup depending on the state
 // provided
-func Freeze(c *cgroups.Cgroup, state cgroups.FreezerState) error {
-	d, err := getCgroupData(c, 0)
+func (m *Manager) Freeze(state cgroups.FreezerState) error {
+	d, err := getCgroupData(m.Cgroups, 0)
 	if err != nil {
 		return err
 	}
 
-	c.Freezer = state
+	m.Cgroups.Freezer = state
 
 	freezer := subsystems["freezer"]
```

Comment:
we should check this and return an error if there is no freezer (I know it's not really related to your PR :) )